example/logger/bugsnag_logger.py

```diff
@@ -0,0 +1,124 @@
+# Initialize Bugsnag to begin tracking errors. Only an api key is required, but here are some other helpful configuration details:
+bugsnag.configure(
+
+    # get your own api key at bugsnag.com
+    api_key = 'YOUR_API_KEY_HERE',
+
+    # if you track deploys or session rates, make sure to set the correct version.
+    app_version = '1.2.3',
+
+    # By default, requests are sent asynchronously. If you would like to block until the request is done, you can set to false
+    asynchronous = False,
+
+    # Defaults to false, this allows you to log each session which will be used to calculate crash rates in your dashboard for each release.
+    auto_capture_sessions = True,
+
+    # Sets which exception classes should never be sent to Bugsnag.
+    ignore_classes = ['Http404', 'DontCare'],
+
+    # Defines the release stage for all events that occur in this app.
+    release_stage = 'development',
+
+    # Defines which release stages bugsnag should report. e.g. ignore staging errors.
+    notify_release_stages = [ 'development', 'production'],
+
+    # Any param key that contains one of these strings will be filtered out of all error reports.
+    params_filters = ["credit_card_number", "password", "ssn"],
+
+    # We mark stacktrace lines as inProject if they come from files inside root:
+    # project_root = "/path/to/your/app",
+
+    # Useful if you are wrapping bugsnag.notify() in with your own library, to ensure errors group properly.
+    # traceback_exclude_module = [myapp.custom_logging],
+
+)
+
+# Create a logger
+logger = logging.getLogger("test.logger")
+
+# Create a Bugsnag handler.
+# Optionally, add 'extra_fields' which will attach meta_data to every Bugsnag report. The values should be attributes to pull off each log record.
+handler = BugsnagHandler(extra_fields={"logger":["__repr__"]})
+
+# Define which level of log you want to report to Bugsnag. Here, warning & above.
+handler.setLevel(logging.WARNING)
+
+# Attach the Bugsnag handler to your logger.
+logger.addHandler(handler)
+
+
+# You can define a callback function which, when attached to your Bugsnag client, will run right before each and every report is sent to the api.  Here you can evaluate and modify the report data.
+def callback(notification):
+    """This callback will evaluate and modify every exception report, handled and unhandled, that occurs within the app, right before it is sent to Bugsnag.
+    """
+    # adding user info and metadata to every report:
+    notification.user = {
+        # in your app, you can pull these details from session.
+        'name': 'Alan Turing',
+        'email': 'turing@code.net',
+        'id': '1234567890'
+    }
+
+    notification.add_tab(
+        'company', {
+            'name': 'Stark Industries'
+        }
+    )
+
+    # The callback will evaluate all exceptions, but in this example only 'SomethingBad' errors will have the below data added to their error reports.
+    if notification.exception.message == 'SomethingBad':
```

Comment:
This is causing an exception to be thrown for me here, `AttributeError: 'MyCustomException' object has no attribute 'message'`